ydb/core/blobstorage/dsproxy/dsproxy_put.cpp

```diff
@@ -42,6 +42,10 @@ class TBlobStorageGroupPutRequest : public TBlobStorageGroupRequestActor {
     TErasureSplitContext ErasureSplitContext = TErasureSplitContext::Init(MaxBytesToSplitAtOnce);
     TBatchedVec<TStackVec<TRope, TypicalPartsInBlob>> PartSets;
 
+    static_assert(MaxBatchedPutRequests <= sizeof(ui64) * 8);
+    std::map<TInstant, ui64> PutDeadlineMasks;
```

Comment:
https://en.cppreference.com/w/cpp/utility/bitset

```suggestion
    std::map<TInstant, std::bitset<MaxBatchedPutRequests>> PutDeadlineMasks;
```

Reply:
please, don't add new asserts unless necessary